exchange/docs-conceptual/exchange-online/connect-to-exchange-online-powershell/mfa-connect-to-exchange-online-powershell.md

```diff
@@ -70,7 +68,7 @@ If you want to use multi-factor authentication (MFA) to connect to Exchange Onli
 
   If basic authentication is disabled, you'll get this error when you try to connect:
 
-  `The WinRM client cannot process the request. Basic authentication is currently disabled in the client configuration. Change the client configuration and try the request again.`
+  > The WinRM client cannot process the request. Basic authentication is currently disabled in the client configuration. Change the client configuration and try the request again.`
 
```

Comment:
It looks like you left a trailing backtick by accident when you converted this from a code block to a quote.

Reply:
I think that originally the article was in this way.

Reply:
I don't think so. The change made above shows that the opening code block enclosure (a single backtick) was replaced with a quote marker (a greater-than sign), but in making such a change the closing code block enclosure (the backtick at the end of that line) should have been removed as well.